rust/src/lexer.rs

```diff
@@ -325,7 +357,7 @@ mod tests {
   fn lex_doc() {
     expect_lex(
       vec!["/// hello"],
-      vec![(0, 0, Symbol::Doc(String::from(" hello")))],
+      vec![(0, 3, Symbol::Doc(String::from(" hello")))],
```

Comment:
I think the position of the docstring should be 0 here not 3.

Reply:
Ack, fixing.